1_fetch/in/Legend_FORESCE_Land_Cover.csv

```diff
@@ -0,0 +1,22 @@
+FORESCE_value,FORESCE_description,Reclassify_match,Reclassify_description
+0,Corn (no data),3,Cultivated Crops
```

Comment:
Corn and no data are labeled as 0? Are the no data values being mapped to cultivated crops?

Reply:
According to the metadata for [historical, current, and future land cover for the Delaware River Basin](https://www.sciencebase.gov/catalog/item/605c987fd34ec5fa65eb6a74) There exists two classifications for corn. 
One is `corn (No Data - Area outside of Delaware River Basin boundary)` - value of 0
The second is `corn` - value of 2

No 0 should 0 classe values should be present in drb. Either we assign it NA or assign it 3 - cultivated crop) as a Ihave done in this table.

Reply:
> One is corn (No Data - Area outside of Delaware River Basin boundary) - value of 0. The second is corn - value of 2

Weird! Do you think the first corn labeled with 0 is a typo and should say "no data" instead of "corn" in the FORESCE metadata? If so, you can report that to the SB database contact

Reply:
possibly? ... probably? We should reach out indeed!

Reply:
Terry got back to me and that is definitely an error. So LULC class 0 should be no data.

Reply:
I' ve edited that line as well.